src/Core/Users/Users.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Meetup\Login\Core;
+
+use Meetup\Login\Core\Users\UserInterface;
+
+class User implements UserInterface
+{
+	public $userName;
+	public $password;
+	public $email;
+	public $createdAt;
+	public $datetime;
+	public $firstName;
+	public $lastName;
+
+	
+	public function setEmail($email)
+	{
+		$this->email = $email;
+	}
+
+	public function getEmail()
+	{
+		return $this->email;
+	}
+
+	public function setPassword($password)
+	{
+		$this->password = $password;
+	}
+
+	public function checkPassword($password)
+	{
+		if($password == $this->password){
+			return true;
+		}
+
+		return false;
+	}	
+
+	public function getId()
+	{
+		$userName = mysqli_real_escape_string($this->userName);
+		$query = "SELECT userId FROM user where username = '{$userName}' and email = '{$email}'";
```

Comment:
Its a getter for a property, why can't it be just `return $this->id;`
No queries are required here. Please remove.